### PR TITLE
issue_2 and issue_6 are done!

### PR DESCRIPTION
**Pull request by AlhaririAnas**
I have created two basic Python files along with their test files for the classes "ant" and "food".

### CODE REVIEW

**Comment by heyitsalina** [comment]
looks good!

File: resources/ant.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+class ant:
```
Comment (honnigmelone) [inline resources/ant.py:2]:
```suggestion
class Ant:
```

File: resources/food.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+class food:
```
Comment (honnigmelone) [inline resources/food.py:2]:
```suggestion
class Food:
```

File: test/test_ant.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from resources.ant import *
```
Comment (honnigmelone) [inline test/test_ant.py:1]:
```suggestion
from resources.Ant import *
```

File: test/test_food.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from resources.food import *
```
Comment (honnigmelone) [inline test/test_food.py:1]:
```suggestion
from resources.Food import *
```

**Review by honnigmelone — CHANGES_REQUESTED** [review]
Would have class-names start with Caps. Apart from that, all looking good,

**Review by honnigmelone — APPROVED** [review]

**Pull request merged**